conform-pr/main.go

```diff
@@ -153,12 +169,11 @@ func (pr *pullRequest) checkBody(action *githubactions.Action) error {
 	// one \n at the end is allowed, but optional
 	match, err := regexp.MatchString(".+[.!?](\n)?$", pr.body)
```

Comment:
If we use `MustCompile` there, this method could only return an error if body is invalid

Reply:
changed.